src/aiida/orm/utils/node.py

```diff
@@ -59,8 +59,8 @@ def load_node_class(type_string):
         except exceptions.MissingEntryPointError:
             return Data
 
-    if base_path.startswith('process'):
-        entry_point_name = base_path.removeprefix('data.')
+    if base_path.startswith('process.'):
+        entry_point_name = base_path.removeprefix('process.')
         return load_entry_point('aiida.node', entry_point_name)
```

Comment:
Based on my local testing, I believe the correct code is this one
```suggestion
        return load_entry_point('aiida.node', base_path)
```